orttraining/tools/scripts/profile/docker/README.md

```diff
@@ -0,0 +1,9 @@
+## Build ORT Docker Images for Philly HyperCluster ##
```

Comment:
Is this applicable to any K8S cluster? If specific to Philly then remove from PR.

Reply:
The whole profile folder is specific to Philly.

Reply:
@pengwa so should we remove orttraining/tools/scripts/profile?

Reply:
I think we can, it is for test purpose. we had it internally, which is enough.